src/jit/lclvars.cpp

```diff
@@ -2676,6 +2676,173 @@ int __cdecl Compiler::WtdRefCntCmp(const void* op1, const void* op2)
     return 0;
 }
 
+#if defined(_TARGET_AMD64_) && defined(FEATURE_CORECLR)
+
+/* static */
+int __cdecl Compiler::WtdRefCntCmpSimpleGC(const void* op1, const void* op2)
+{
```

Comment:
Please add a new standard-format function header.

You don't care about weighted ref counts in this case, right?, so why is it named "Wtd..."?

In fact, couldn't you assert:
```
assert(!dsc1->lvTracked);
assert(!dsc2->lvTracked);
```
?

So the only thing you are doing is sorting GC refs together?